tests/test_eload_preparation.py

```diff
@@ -99,7 +100,20 @@ def test_find_genome_single_sequence(self):
         cfg.content['eutils_api_key'] = None
         self.eload.eload_cfg.set('submission', 'scientific_name', value='Thingy thingus')
         self.eload.eload_cfg.set('submission', 'analyses', 'Analysis alias test', 'assembly_accession', value='AJ312413.2')
-        self.eload.find_genome()
-        assert self.eload.eload_cfg['submission']['analyses']['Analysis alias test']['assembly_fasta'] \
-               == 'tests/resources/genomes/thingy_thingus/AJ312413.2/AJ312413.2.fa'
-        assert 'assembly_report' not in self.eload.eload_cfg['submission']
+
+        with mock.patch("eva_submission.eload_preparation.requests.put", return_value=Mock(status_code=200)):
+            self.eload.find_genome()
+            assert self.eload.eload_cfg['submission']['analyses']['Analysis alias test']['assembly_fasta'] \
+                   == 'tests/resources/genomes/thingy_thingus/AJ312413.2/AJ312413.2.fa'
+            assert 'assembly_report' not in self.eload.eload_cfg['submission']
+
+    def test_contig_alias_db_update(self):
+
+        cfg.content['eutils_api_key'] = None
+        self.eload.eload_cfg.set('submission', 'scientific_name', value='Thingy thingus')
+        self.eload.eload_cfg.set('submission', 'analyses', 'Analysis alias test', 'assembly_accession',
+                                                   value='AJ312413.2')
```

Comment:
This is being mocked so maybe not a real problem but you're passing a sequence not an assembly here. Replace with a GCA.

Reply:
Although you're not mocking the retrieval of the fasta sequence so if you set an assembly it will download the fasta and assembly report which would be slow.
Probably best to mock that as well.

Reply:
This is incorporated with the new changes